membership-attribute-service/test/acceptance/AccountControllerAcceptanceTest.scala

```diff
@@ -49,7 +49,7 @@ class AccountControllerAcceptanceTest extends AcceptanceTest {
     contactRepositoryMock = mock[ContactRepository]
     subscriptionServiceMock = mock[SubscriptionService[Future]]
     zuoraRestServiceMock = mock[ZuoraRestService]
-    catalogServiceMock = TestCatalog()
+    catalogServiceMock = catalog
```

Comment:
now we use the TestCatalog from membership-common tests instead of the blank dummy one, as the code needs to access the catalog in the business logic in order to get some of the information